cpp/src/Ice/Selector.h

```diff
@@ -15,10 +15,6 @@
 #    include <sys/epoll.h>
 #elif defined(ICE_USE_KQUEUE)
 #    include <sys/event.h>
-#elif defined(ICE_USE_IOCP)
-// Nothing to include
-#elif defined(ICE_USE_POLL)
-#    include <sys/poll.h>
 #endif
 
 #include <condition_variable>
```

Comment:
I would move condition_variable next to map and add ICE_USE_CFSTREAM in an elif.